index.html

```diff
@@ -2695,12 +2695,12 @@ <h3>Processing Errors</h3>
 The `verificationMethod` value in a [=proof=] was malformed. See Section
 [[[#retrieve-verification-method]]].
           </dd>
-          <dt id="INVALID_CONTROLLER_DOCUMENT_ID">INVALID_CONTROLLER_DOCUMENT_ID (-22)</dt>
+          <dt id="INVALID_CONTROLLED_IDENTIFIER_DOCUMENT_ID">INVALID_CONTROLLED_IDENTIFIER_DOCUMENT_ID (-22)</dt>
```

Comment:
Note: I think we removed integers for the errors in the other specs, if so, we should be consistent and remove those here as well. cc: @msporny

Reply:
I agree, but maybe cleaner if it is done in a separate PR after this one is merged.